src/platforms/Token/EditTokenModal.tsx

```diff
@@ -106,6 +107,12 @@ const EditTokenModal = ({
   const { data: requirements } = useRequirements(role.id)
   const snapshotRequirement = requirements?.find((req) => !!req?.data?.snapshot)
 
+  const { captureEvent } = usePostHogContext()
```

Comment:
If we won't allow users to add this reward in the edit role drawer, we don't need events here ;)

Reply:
But they can be edited through the access hub card, are you sure?

Reply:
Oh, I forgot about that. You're right, then let's leave these events here! 👌